### PR TITLE
[DOCS] Remove, relocate, consolidate, and edit Contributing content

### PR DESCRIPTION
**Pull request by kwcanuck**
### What is the issue?

Currently, there is a _[Contributing](https://docs.greatexpectations.io/docs/contributing/contributing)_ section in the Great Expectations help. When I started reviewing this content, I noticed information was duplicated, content was out of date, and the organizational structure was confusing. In addition, research and analysis indicates that this type of content is typically provided in GitHub repositories and not within product help. 

### What was done?

- Consolidation of the code contribution process into a single document named _[Contribute a code change](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md)_

- Consolidation of the package contribution process into a single document named _[Contribute packages](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_PACKAGES.md)_

- Consolidation of the Expectation contribution process into a single document named _[Contribute custom Expectations](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md)_ 

- Consolidation of the documentation contribution process into a single document titled _[Contribute to Great Expectations documentation](https://github.com/great-expectations/great_expectations/tree/contributing-and-style-readme/docs)_

- Removing the topics used to create the new  _Contribute a code change_, _Contribute packages_, and _Contribute custom Expectations_ topics from the existing TOC 

- Removing the topics used to create the new _Contribute to Great Expectations documentation_ from the existing TOC

- Creating a new README for the _Contribute to Great Expectations documentation_ content and adding the README to the docs repository so it is the first item users see

- Adding the content from the _How to write a how-to-guide_ and _How to write integration documentation_ topics to  _Contribute to Great Expectations documentation_ 

- Creating a new `template` folder in the docs repository and adding the how-to and integration templates

- Renaming the existing _Levels of Maturity_ topic  _[Feature and code readiness](https://deploy-preview-7669--niobium-lead-7998.netlify.app/docs/contributing/contributing_maturity)_ and moving the topic to the _Reference_ section

- Adding a _Contribute_ topic to the TOC below _Welcome_

- Performing an initial edit of all content and correcting style, grammar, and consistency issues

- Removing the existing _Contributions_ section from the TOC

CC: @joshzzheng

### CODE REVIEW

**Review by joshzzheng — APPROVED** [review]

**Review by tjholsman — CHANGES_REQUESTED** [review]
This change removes most of the existing Contributing sidebar in the docs but it does not remove all of the docs themselves. Is that intended?

One related example, [Creating custom expectations](https://deploy-preview-7669--niobium-lead-7998.netlify.app/docs/guides/expectations/creating_custom_expectations/overview) has a link near the end to [How to contribute a custom expectation](https://deploy-preview-7669--niobium-lead-7998.netlify.app/docs/guides/expectations/contributing/how_to_contribute_a_custom_expectation_to_great_expectations), which has been removed from sidebar but not removed from the docs in this PR. This renders a standalone docs page with no sidebar. Should this be updated to point to the new Github page CONTRIBUTING_EXPECTATIONS?

**Comment by kwcanuck** [comment]
@tjholsman you are correct. This PR removes the Contributing section (Sidebar) entirely. I have not deleted the individual topics as I wasn't sure if this PR would be approved as submitted. If a reviewer requested that an original topic be returned to the TOC, it would be a quick fix if the topic was still available. I'll remove the topics that are no longer used when the PR is approved. 

I'll also be correcting any links and/or adding a redirect for any topics that are removed. It's all part of the final clean up before pushing the changes. Again, I didn't want to go too far in case my changes were not accepted as submitted. 

For your reference, here are the redirects:

image

Here is the commit that removed the unused topics:

image

**Review by tjholsman — APPROVED** [review]
lgtm - thanks, @kwcanuck!

File: docs/docusaurus/sidebars.js [hunk 1]
```diff
@@ -442,75 +442,16 @@ module.exports = {
...
       items: [
-        {
-          type: 'category',
-          label: 'Supplemental documentation',
-          link: { type: 'doc', id: 'reference/supplemental_documentation' },
-          items: [
-            { type: 'doc', id: 'guides/miscellaneous/how_to_use_the_great_expectations_cli' },
-            { type: 'doc', id: 'guides/miscellaneous/how_to_use_the_project_check_config_command' },
-            { type: 'doc', id: 'reference/customize_your_deployment' },
-            { type: 'doc', id: 'reference/usage_statistics' }
```
Comment (Rachel-Reverie) [inline docs/docusaurus/sidebars.js:511]: Relocating this may require approval from outside of DevRel.

File: docs/docusaurus/static/_redirects [hunk 2]
```diff
@@ -250,4 +250,18 @@
...
-/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
\ No newline at end of file
+/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
+
+# Redirects for Contributing docs
+
+/docs/contributing/contributing_setup https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md
+/docs/contributing/contributing_checklist /docs/contributing/contributing
+/docs/contributing/contributing_github /docs/README
+/docs/contributing/contributing_test /ghttps://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md
```
Comment (Rachel-Reverie) [inline docs/docusaurus/static/_redirects]:
```suggestion
/docs/contributing/contributing_test /https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md
```
Pretty sure that extra `g` was a typo.

File: docs/docusaurus/docs/templates/how_to_template.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 title: TEMPLATE How to guide {stub}
 ---
-import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx'
-import NextSteps from '../connecting_to_your_data/components/next_steps.md'
+import Prerequisites from '/docs/guides/connecting_to_your_data/components/prerequisites.jsx'
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/templates/how_to_template.md]:
```suggestion
import Prerequisites from '/docs/components/_prerequisites.jsx'
```
This is an old link.  We're trying to consolidate "universal" components like this in `/docs/components/` so we can more easily update formatting for them.

The same applies to the "NextSteps" import, except that I don't believe we have an updated version in `/docs/components/` yet.

File: docs/docusaurus/static/_redirects [hunk 4]
```diff
@@ -250,4 +250,18 @@
...
 # removing yml-driven datacontext in favor of ephemeral
-/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
\ No newline at end of file
+/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
+
+# Redirects for Contributing docs
+
+/docs/contributing/contributing_setup https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md
+/docs/contributing/contributing_checklist /docs/contributing/contributing
+/docs/contributing/contributing_github /docs/README
```
Comment (Rachel-Reverie) [inline docs/docusaurus/static/_redirects]: I tested this redirect in the netlify preview and it does not appear to be working.

**Review by Rachel-Reverie — APPROVED** [review]

File: docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md [hunk 5]
```diff
@@ -316,7 +316,7 @@ Congratulations!<br/>&#127881; You've just built your first Custom Expectation!
 
 ### 9. Contribution (Optional)
 
-This guide will leave you with a Custom Expectation sufficient for [contribution](../contributing/how_to_contribute_a_custom_expectation_to_great_expectations.md) back to Great Expectations at an Experimental level.
+This guide will leave you with a Custom Expectation sufficient for contribution to Great Expectations at an Experimental level.
```
Comment (austiezr) [inline docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md]:
```suggestion
This guide will leave you with a Custom Expectation sufficient for [contribution](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) to Great Expectations at an Experimental level.
```

File: CONTRIBUTING_CODE.md [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+
+- [Installing Microsoft ODBC driver for Linux](https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server)
+
+
+##  Install Great Expectations
+
+Run the following command to install Great Expectations in editable mode:
+
+```sh
+pip install -e
```
Comment (austiezr) [inline CONTRIBUTING_CODE.md]:
```suggestion
pip install -e .
```

Comment (austiezr) [reply CONTRIBUTING_CODE.md]: May be worth noting that this must be run from the root of the GX repo; not a strongly held opinion though.

File: CONTRIBUTING_EXPECTATIONS.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
+# Contribute custom Expectations
+
+To submit a custom Expectation to Great Expectations for consideration, you complete the following tasks:
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
To submit a Custom Expectation to Great Expectations for consideration, you complete the following tasks:
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+To create and submit a custom package to Great Expectations for consideration, see [CONTRIBUTING_PACKAGES](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_PACKAGES.md) in the `great_expectations` repository.
+
+To submit a code change to Great Expectations for consideration, see [CONTRIBUTING_CODE](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md) in the `great_expectations` repository.
+
+## Prerequisites
+
+- Great Expectations installed and configured for your environment. See [Great Expectations Quickstart](https://docs.greatexpectations.io/docs/tutorials/quickstart/).
+
+- A custom Expectation. See [Creating Custom Expectations](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/overview/).
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
- A Custom Expectation. See [Creating Custom Expectations](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/overview/).
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+    ```
+6. Run the following command to create a branch for your changes:
+
+    ```sh
+    git checkout -b <branch-name>
+    ```
+
+## Generate the Expectation validation checklist
+
+Before you submit your custom Expectation, you need to verify it meets the submission requirements. Great Expectations provides a checklist to help you determine if your custom Expectation meets the minimum requirements. Your custom Expectation must meet five of the listed requirements to qualify for submission.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
Before you submit your Custom Expectation, you need to verify it meets the submission requirements. Great Expectations provides a checklist to help you determine if your Custom Expectation meets the minimum requirements. Your Custom Expectation must meet the first five of the listed requirements to qualify for submission.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+To generate the Expectation checklist, add the `print_diagnostic_checklist()` instance method to your custom Expectation. When the instance method runs, it returns results similar to the following:
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
To generate the Expectation checklist, add the `print_diagnostic_checklist()` instance method to your Custom Expectation. When the instance method runs, it returns results similar to the following:
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+✔ Has both statement Renderers: prescriptive and diagnostic
+✔ Has core logic that passes tests for all applicable Execution Engines and SQL dialects
+  ...
+  Has a full suite of tests, as determined by project code standards
+  Has passed a manual review by a code owner for code standards and style guides
+```
+
+## Verify Expectation metadata
+
+Verifying your custom Expectation metadata ensures that it is accredited to you and includes an accurate description.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
Verifying your Custom Expectation metadata ensures that it is accredited to you and includes an accurate description.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+Great Expectations maintains a number of custom Expectation packages, that contain thematically related custom Expectations. These packages are located in the [Great Expectations contrib directory](https://github.com/great-expectations/great_expectations/tree/develop/contrib) and on [PyPI](https://pypi.org/).
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
Great Expectations maintains a number of Custom Expectation packages, that contain thematically related Custom Expectations. These packages are located in the [Great Expectations contrib directory](https://github.com/great-expectations/great_expectations/tree/develop/contrib) and on [PyPI](https://pypi.org/).
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+If your custom Expectation fits within one of these packages, you're encouraged to contribute your custom Expectation directly to one of these packages.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
If your Custom Expectation fits within one of these packages, you're encouraged to contribute your Custom Expectation directly to one of these packages.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+
+If you're not contributing to a specific package, your Custom Expectation is automatically published in the [PyPI great-expectations-experimental package](https://pypi.org/project/great-expectations-experimental/). This package contains all Great Expectations experimental community-contributed custom Expectations that have not been submitted to other packages.
+
+Confirm the `library_metadata` object for your custom Expectation includes the following information:
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
Confirm the `library_metadata` object for your Custom Expectation includes the following information:
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+## Verify Expectation metadata
+
+Verifying your custom Expectation metadata ensures that it is accredited to you and includes an accurate description.
+
+Great Expectations maintains a number of custom Expectation packages, that contain thematically related custom Expectations. These packages are located in the [Great Expectations contrib directory](https://github.com/great-expectations/great_expectations/tree/develop/contrib) and on [PyPI](https://pypi.org/).
+If your custom Expectation fits within one of these packages, you're encouraged to contribute your custom Expectation directly to one of these packages.
+
+If you're not contributing to a specific package, your Custom Expectation is automatically published in the [PyPI great-expectations-experimental package](https://pypi.org/project/great-expectations-experimental/). This package contains all Great Expectations experimental community-contributed custom Expectations that have not been submitted to other packages.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
If you're not contributing to a specific package, your Custom Expectation is automatically published in the [PyPI great-expectations-experimental package](https://pypi.org/project/great-expectations-experimental/). This package contains all Great Expectations experimental community-contributed Custom Expectations that have not been submitted to other packages.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+
+Confirm the `library_metadata` object for your custom Expectation includes the following information:
+
+- `contributors`: Identifies the creators of the custom Expectation.
+- `tags`: Identifies the custom Expectation functionality and domain. For example, `statistics`, `flexible comparisons`, `geography`, and so on.
+- `requirements`: Identifies if your custom Expectation relies on third-party packages.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
- `contributors`: Identifies the creators of the Custom Expectation.
- `tags`: Identifies the Custom Expectation functionality and domain. For example, `statistics`, `flexible comparisons`, `geography`, and so on.
- `requirements`: Identifies if your Custom Expectation relies on third-party packages.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+
+3. Add a meaningful title and description for your pull request (PR). Provide a detailed explanation of what you changed and why. To help identify the type of issue you’re submitting, add one of the following identifiers to the pull request (PR) title:
+
+    - [BUGFIX] for PRs that address minor bugs without changing behavior.
+
+    - [FEATURE] for significant PRs that add a new feature likely to require being added to our feature maturity matrix.
+
+    - [MAINTENANCE] for PRs that focus on updating repository settings or related changes.
+		
+    - [CONTRIB] for the contribution of custom Expectations and supporting work into the `contrib/` directory.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
    - [CONTRIB] for the contribution of Custom Expectations and supporting work into the `contrib/` directory.
```

File: CONTRIBUTING_CODE.md [hunk 18]
```diff
@@ -0,0 +1,514 @@
...
+
+- [Test performance](#test-performance)
+
+- [Submit a pull request](#submit-a-pull-request)
+
+To discuss your code change before you implement it, join the [Great Expectations Slack community](https://greatexpectations.io/slack) and make your suggestion in the [#contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) channel. 
+
+To request a documentation change, or a change that doesn't require local testing, see the [README](https://github.com/great-expectations/great_expectations/tree/contributing-and-style-readme/docs) in the `docs` repository.
+
+To create and submit a custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```
Comment (austiezr) [inline CONTRIBUTING_CODE.md]:
```suggestion
To create and submit a Custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```

File: CONTRIBUTING_CODE.md [hunk 19]
```diff
@@ -0,0 +1,514 @@
...
+    git remote add upstream git@github.com:great-expectations/great_expectations.git
+    ```
+6. Run the following command to create a branch for your changes:
+
+    ```sh
+    git checkout -b <branch-name>
+    ```
+## Install Python dependencies
+
+Python dependencies are required to modify Great Expectations code, submit a new feature, or submit a custom Expectation.
```
Comment (austiezr) [inline CONTRIBUTING_CODE.md]:
```suggestion
Python dependencies are required to modify Great Expectations code, submit a new feature, or submit a Custom Expectation. 
```

File: CONTRIBUTING_CODE.md [hunk 20]
```diff
@@ -0,0 +1,514 @@
...
+
+3. Add a meaningful title and description for your pull request (PR). Provide a detailed explanation of what you changed and why.  To help identify the type of issue you’re submitting, add one of the following identifiers to the pull request (PR) title:
+
+    - [BUGFIX] for PRs that address minor bugs without changing behavior.
+
+    - [FEATURE] for significant PRs that add a new feature likely to require being added to our feature maturity matrix.
+
+    - [MAINTENANCE] for PRs that focus on updating repository settings or related changes.
+		
+    - [CONTRIB] for the contribution of custom Expectations and supporting work into the `contrib/` directory.
```
Comment (austiezr) [inline CONTRIBUTING_CODE.md]:
```suggestion
    - [CONTRIB] for the contribution of Custom Expectations and supporting work into the `contrib/` directory.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
+# Contribute custom Expectations
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
# Contribute Custom Expectations
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+
+4. If this is your first Great Expectations contribution, you'll be prompted to complete the Contributor License Agreement (CLA). Complete the CLA and add `@cla-bot check` as a comment to the pull request (PR) to indicate that you’ve completed it.
+
+5. Wait for the Continuous Integration (CI) checks to complete and then correct any syntax or formatting issues.
+
+    A Great Expectations team member reviews, approves, and merges your PR. Depending on your GitHub notification settings, you'll be notified when there are comments or when your changes are successfully merged.
+
+    If your custom Expectation doesn't meet the minimum requirements in the validation checklist, it is failing testing, or there is a functionality error, you'll be asked to resolve the issues before your custom Expectation can move forward.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
    If your Custom Expectation doesn't meet the minimum requirements in the validation checklist, it is failing testing, or there is a functionality error, you'll be asked to resolve the issues before your Custom Expectation can move forward.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+
+    If you are submitting a production custom Expectation, Great Expectations requires that your custom Expectation meet or exceed Great Expectation standards for testing and coding.
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
    If you are submitting a production Custom Expectation, Great Expectations requires that your Custom Expectation meet or exceed Great Expectation standards for testing and coding.
```

File: docs/docusaurus/docs/contributing/contributing_overview.md [hunk 24]
```diff
@@ -1,36 +1,19 @@
...
+To contribute to Great Expectations documentation or code, see one of the following resources:
 
-Our goal is to make your experience as great as possible. Please follow these steps to contribute:
+- To request a documentation change, or a change that doesn't require local testing, see the [README](https://github.com/great-expectations/great_expectations/tree/contributing-and-style-readme/docs) in the `docs` repository.
 
-#### 1. Join the community on Slack
+- To submit a code change to Great Expectations for consideration, see [CONTRIBUTING_CODE](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md) in the `great_expectations` repository.
 
-* Go to [greatexpectations.io/slack](https://greatexpectations.io/slack) and introduce yourself in the [#contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) channel.
+- To create and submit a custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```
Comment (austiezr) [inline docs/docusaurus/docs/contributing/contributing_overview.md]:
```suggestion
- To create and submit a Custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```

File: CONTRIBUTING_EXPECTATIONS.md [hunk 25]
```diff
@@ -0,0 +1,136 @@
...
+
+    When your custom Expectation has successfully passed testing and received approval from a code owner, your contribution is complete. Your custom Expectation will be included in the next release of Great Expectations and an announcement will appear in the release notes
```
Comment (austiezr) [inline CONTRIBUTING_EXPECTATIONS.md]:
```suggestion
    When your Custom Expectation has successfully passed testing and received approval from a code owner, your contribution is complete. Your custom Expectation will be included in the next release of Great Expectations and an announcement will appear in the release notes
```

File: CONTRIBUTING_PACKAGES.md [hunk 26]
```diff
@@ -0,0 +1,155 @@
...
+
+To create and submit a custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
+
+To submit a code change to Great Expectations for consideration, see [CONTRIBUTING_CODE](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md) in the `great_expectations` repository.
+
+## Prerequisites
+
+- Great Expectations installed and configured for your environment. See [Great Expectations Quickstart](https://docs.greatexpectations.io/docs/tutorials/quickstart/).
+
+- A custom Expectation. See [Creating Custom Expectations](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/overview/).
```
Comment (austiezr) [inline CONTRIBUTING_PACKAGES.md]:
```suggestion
- A Custom Expectation. See [Creating Custom Expectations](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/overview/).
```

File: CONTRIBUTING_PACKAGES.md [hunk 27]
```diff
@@ -0,0 +1,155 @@
...
+
+- [Initialize a project](#initialize-a-project)
+
+- [Contribute to your package](#contribute-to-your-package)
+
+- [Publish your package](#publish-your-package)
+
+To request a documentation change, or a change that doesn't require local testing, see the [README](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/docs/README.md) in the `docs` repository.
+
+To create and submit a custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```
Comment (austiezr) [inline CONTRIBUTING_PACKAGES.md]:
```suggestion
To create and submit a Custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_EXPECTATIONS.md) in the `great_expectations` repository.
```

**Review by austiezr — CHANGES_REQUESTED** [review]
1 necessary fix re: installing from the repo.

Other changes align with previous convention but not strongly held and won't gate.

**Review by austiezr — APPROVED** [review]
🚀

**Pull request merged**